### DISCUSSION TITLE
New Future Action: Change Visibility

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-future#560] [Feature Requests]
New Future Action: Change Visibility

I would like to extend the deadline of an article whose visibility is Private to a future time. Is there a way to automatically change Visibility from Private to Public at that time? Is there no choice but to change it manually?

Original Request: https://wordpress.org/support/topic/is-it-possible-to-automatically-switch-visibility/

**Comment by andergmartins** [comment]
We've implemented this for workflows by using the step "Update post".
